pysrc/bytewax/connectors/kafka/serde.py

```diff
@@ -0,0 +1,94 @@
+"""Serializers and deserializers for kafka messages."""
+import io
+import json
+import logging
+from abc import ABC, abstractmethod
+
+from confluent_kafka.schema_registry.avro import AvroDeserializer, AvroSerializer
+from confluent_kafka.serialization import (
+    MessageField,
+    SerializationContext,
+)
+from fastavro import parse_schema, schemaless_reader, schemaless_writer
+
+logger = logging.getLogger(__name__)
+
+
+class SerializationError(ValueError):
+    """(de)serialization exception."""
+
+
+class SchemaSerializer(ABC):
+    """A serializer for a specific schema."""
+
+    @abstractmethod
+    def ser(self, obj: dict, topic: str) -> bytes:
```

Comment:
Yep, this was here to resemble confluent's api, but in our case the topic shouldn't change between messages, so I'm going to remove this  argument from here.
But the way it's implemented now, the serde interface still requires `self` to avoid reinitializing (de)serializers at each message, so not exactly the same as the one we already have